### DISCUSSION TITLE
Assistance: Salzburg, Northern Sky, 425 BC

### DISCUSSION THREAD

**Discussion by karimmata** [Stellarium/stellarium#1765] [Q&A]
Hello,
I was wondering if this would be the place to ask for assistance from someone who has the software and knows how to use it.
As per the title, I would be very interested in discovering what the northern night sky looked like (equinox and solstice) in 425 BC. Were constellations in close to current place or is there a significant difference from their current location?
Thank you!
Best,
Karim

**Comment by gzotti** [comment]
The great thing with Stellarium, and one of the reasons for me to keep improving it, is that you can simply look this up yourself. Remaining accuracy issues are discussed in the User Guide.
